tests/server/blueprints/institutes/test_institute_views.py

```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 from flask import url_for
 from scout.server.extensions import store
+import logging
```

Comment:
I think you can remove the import and the debug LOG message from this test. It won't be seen anyway

Reply:
Done!

Reply:
> I think you can remove the import and the debug LOG message from this test. It won't be seen anyway

`pytest --log-cli-level=10` 🤓

Reply:
Nice!